src/components/UI/FetchError/index.tsx

```diff
@@ -5,6 +5,20 @@ const refreshPage = (): void => {
   window.location.reload();
 };
 
+const getHumanReadableErrorMessages = (error: ErrorFromFetch): string => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const errorStatusCode = error.error.response.status;
+  switch (errorStatusCode) {
+    case 404:
+      return '404: The requested resource could not be found.';
+    case 500:
+      return '500: The server encountered an error.';
+    default:
+      return 'An error occurred.';
```

Comment:
```suggestion
      return `${errorStatusCode}: An error occurred.`;
```
(otherwise I don't think the code for e.g. a 403 or 502 would be rendered)